api/fossa/tar.go

```diff
@@ -136,11 +139,12 @@ func UploadTarball(dir string) (Locator, error) {
 		return Locator{}, err
 	}
 
-	return Locator{
+	locator := Locator{
```

Comment:
This change seems like a no-op.

Reply:
You're right, I did this so I could print the locator before returning.